docker/entrypoint.d/56-network-driver-version-check.sh

```diff
@@ -1 +1 @@
-
+#!/bin/bash
```

Comment:
Could you please clarify to me why we need `entrypoint.d` files? And are we keeping them empty, or we are adding `#!/bin/bash` to them? I think we need to be consistent

Reply:
I don't know, this would be a better question for @cliffwoolley who made the change. This pull request is centered around formatting exclusively, so executable files should have shebangs. That should not change its core functionality.

I appreciate you reviewing this PR. It might make sense to pause work on this pull request. We're discussing doing a rollout in waves. Since this is the biggest code change, it'd probably be in the last wave and given all the rebasing it'll need, it'll probably need a fresh review then (sorry!). The alternative is trying to get it in now/first, but I think we'd be hard-pressed to make that happen without dedicated reviewer time (e.g. getting you and any other reviewers tickets to review this, given its size).

Reply:
No worries. I thought initial screen could speed up future work, since found issues would still need to be fixed. I'll pause for now.

Reply:
The reason for these to be here is to override the files of the same name in our base image.  Please don't change them.  They are sourced rather than executed directly so the shebang is irrelevant.

Reply:
Thanks for clarifying, Cliff. And thank you Olga for inquiring. Reverted these changes.

Reply:
Olga, feel free to resume reviewing if you find time. No worries if you can't, especially given the size of this PR. We got most of the other repos in and I realized that there's a pretty easy and safe way to rebase this (wherever there are merge conflicts, we can just use the current server repo file and re-run the hooks), confirmed by CI continuing to pass. So figure we can work to get this in.

Also, it might be worthwhile to create a JIRA ticket to account for the time you're working on this.